cmd/daemon/main.go

```diff
@@ -42,11 +42,12 @@ import (
 )
 
 var (
-	bindAddr         = flag.String("addr", ":10221", "The address the metric endpoint and healthz binds to.")
-	pprofAddr        = flag.String("pprof-addr", ":10222", "The address the pprof binds to.")
-	enablePprof      = flag.Bool("enable-pprof", true, "Enable pprof for daemon.")
-	pluginConfigFile = flag.String("plugin-config-file", "/kruise/CredentialProviderPlugin.yaml", "The path of plugin config file.")
-	pluginBinDir     = flag.String("plugin-bin-dir", "/kruise/plugins", "The path of directory of plugin binaries.")
+	bindAddr               = flag.String("addr", ":10221", "The address the metric endpoint and healthz binds to.")
+	pprofAddr              = flag.String("pprof-addr", ":10222", "The address the pprof binds to.")
+	enablePprof            = flag.Bool("enable-pprof", true, "Enable pprof for daemon.")
+	pluginConfigFile       = flag.String("plugin-config-file", "/kruise/CredentialProviderPlugin.yaml", "The path of plugin config file.")
+	pluginBinDir           = flag.String("plugin-bin-dir", "/kruise/plugins", "The path of directory of plugin binaries.")
+	maxWorkersForPullImage = flag.Int("max-workers-for-pull-image", -1, "The maximum number of workers for pulling images.")
```

Comment:
Add todo, After the feature is stable, the default value should also be restricted, e.g. 5.